libs/error-analysis/src/lib/ErrorAnalysisDashboard/Controls/MainMenu/MainMenu.tsx

```diff
@@ -91,11 +91,11 @@ export class MainMenu extends React.PureComponent<IMainMenuProps> {
         {
           commandBarButtonAs: (): any => (
             <Label styles={labelStyle}>
-              {localization.ErrorAnalysis.MainMenu.errorDetectorLabel}
+              {localization.ErrorAnalysis.MainMenu.errorExplorerLabel}
             </Label>
           ),
-          key: "errorDetectorLabel",
-          text: "Error Detector Label"
+          key: "errorExplorerLabel",
+          text: "Error explorer label"
```

Comment:
this should be localized I think (?)

Reply:
it's not shown anywhere I believe?

Reply:
... in which case: why is there a label?

Reply:
the commandBarButtonAs overrides it